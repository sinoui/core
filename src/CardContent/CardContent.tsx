import styled from 'styled-components';

const CardContent = styled.div`
  padding: 16px;
  color: ${(props) => props.theme.palette.text.primary};
  box-sizing: border-box;
`;

export default CardContent;
