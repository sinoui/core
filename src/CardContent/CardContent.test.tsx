import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from 'styled-components';
import { defaultTheme } from '@sinoui/theme';
import CardContent from './CardContent';

describe('快照测试', () => {
  it('CardContent', () => {
    const tree = renderer
      .create(
        <ThemeProvider theme={defaultTheme}>
          <CardContent />
        </ThemeProvider>,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
