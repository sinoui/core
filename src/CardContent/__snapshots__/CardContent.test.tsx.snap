// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`快照测试 CardContent 1`] = `
<div
  className="sc-bdvvtL kGnISe"
/>
`;
